packages/mirrorful/editor/src/pages/index.tsx

```diff
@@ -1,8 +1,38 @@
 import { Dashboard } from '@mirrorful/core/lib/components/Dashboard'
+import ServerEndedMessage from '@mirrorful/core/lib/components/ServerEndedMessage'
 import { TConfig } from '@mirrorful/core/lib/types'
 import Head from 'next/head'
+import { useState, useRef, useEffect } from 'react'
 
 export default function Editor() {
+  const [hasShutDown, setHasShutDown] = useState(false)
+  const counter = useRef<boolean>(false)
+  useEffect(() => {
+    function pollForServerEndCheck() {
+      fetch('http://localhost:5050/api/longPoll', { keepalive: true })
```

Comment:
Hmm.. the only tricky thing is in the future we wanted to allow the user to pass their own port, and so hardcoding to be 5050 I'm not sure about

Reply:
Yeah, we can make it so that it accepts a port in the future. But we would need to know where that variable for the port is located.

Reply:
Made Url and port into their own variable for now.